### ISSUE TITLE
Update the focus outline on buttons to increase the contrast to improve accessibility

### ISSUE THREAD

**Issue by caendesilva** [caendesilva/mutual-aid-app#6]
As it is now, the focus outline on buttons has very low contrast, see attached screenshot. This makes it difficult for sighted keyboard navigating users. The contrast should be increased (or opacity decreased) to improve accessibility.

![image](https://user-images.githubusercontent.com/95144705/157937152-d5bb62d8-1663-4c61-90a4-91bcad5df80d.png)
Both buttons are in the focus state.
![image](https://user-images.githubusercontent.com/95144705/157937220-30493c18-c0e8-49f5-bd17-dfbf455ed8ac.png)
Neither of the buttons are focused.

**Comment by emmadesilva** [comment]
Closing as the project is being archived. https://github.com/caendesilva/mutual-aid-app/issues/48
